### ISSUE TITLE
Theme Accessibility - Lesson Plan

### ISSUE THREAD

**Issue by courtneyr-dev** [WordPress/Learn#360]
https://learn.wordpress.org/lesson-plan/theme-accessibility/
https://github.com/wptrainingteam/theme-accessibility

# Topic Description
Write a description of the topic here.

# Related Resources
Links to related content on Learn, HelpHub, DevHub, GitHub Gutenberg Issues, DevNotes, etc.

- 

# Next steps for SMEs
- [ ] Assign a priority
- [ ] Write out 2-3 learning objectives
- [ ] Change `Status` to `Vetted Topics`

**Comment by annezazu** [comment]
Want to quickly note some of the complexities surrounding this with block themes: https://github.com/WordPress/gutenberg/issues/39266 Shouldn't be a blocker but should be noted and it's likely that a11y SME will need to be brought in.

**Comment by kaitohm** [comment]
Closing, due to not enough information. (See [this Slack conversation](https://wordpress.slack.com/archives/C03B11FA16D/p1688486841217239?thread_ts=1687827128.413619&cid=C03B11FA16D) for more information.)

If you'd still like to see this content created, please create [a new Topic Idea issue](https://wordpress.slack.com/archives/C03B11FA16D/p1688486841217239?thread_ts=1687827128.413619&cid=C03B11FA16D) and fill out the information listed.
